File: GymCRM.SchedulingAPI/Models/DTOs/TrainingSession.cs
namespace GymCRM.SchedulingAPI.Models.DTOs;

public class TrainingSession
{
    public Guid Id { get; set; }
    public Guid TrainerId { get; set; }
    public Guid ClientId { get; set; }
    public DateTime StartTime { get; set; }
    public DateTime EndTime { get; set; }
    public int Status { get; set; }
    public string? Description { get; set; }
}

File: gymcrm.web/src/modules/scheduling/types/trainingSession.ts
// Mirrors GymCRM.SchedulingAPI.Models.DTOs.TrainingSession 1:1.
// trainerId/clientId are the trainer's/member's AccountGuid, not Member.Id
// (Member has no Id field - only accountGuid - see types/member.ts).
// startTime/endTime are naive local datetime strings ("YYYY-MM-DDTHH:mm:ss") -
// no timezone conversion happens anywhere in this module.
export interface TrainingSession {
    id: string;
    trainerId: string;
    clientId: string;
    startTime: string;
    endTime: string;
    status: number;
    description?: string;
}
